bolt/lib/Core/DebugData.cpp

```diff
@@ -395,6 +393,13 @@ DebugAddrWriter::DebugAddrWriter(BinaryContext *BC) : BC(BC) {
   AddressStream = std::make_unique<raw_svector_ostream>(*Buffer);
 }
 
+DebugAddrWriter::DebugAddrWriter(BinaryContext *BC,
+                                 const uint8_t AddressByteSize)
+    : BC(BC), AddressByteSize(AddressByteSize) {
+  Buffer = std::make_unique<AddressSectionBuffer>();
```

Comment:
Remove redundant code, either call this constructor from :DebugAddrWriter(BinaryContext *BC) with some invalid AddressByteSize, or factor out to private "init" function.